src/utils/interfaces.ts

```diff
@@ -214,3 +215,22 @@ export interface IDimension {
     width: number;
     height: number;
 }
+
+export interface IBubblerProcessParamObj {
+    optValue: number,
+    value: number | undefined
+}
+
+export interface IBubblerProcessParam {
+    bubbleObj: IBubblerProcessParamObj,
+    particlesObj: IBubblerProcessParamObj,
+    type: ProcessBubbleType
+}
+
+export interface IShapeSide {
+    length: number,
+    count: {
+        numerator: number,
+        denominator: number,
+    }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5012291)